docs/profiler/profiling.md

```diff
@@ -0,0 +1,438 @@
+- Lead:Michael Mauderer
+- Start Date: 2021-11-15
```

Comment:
Let's remove these from docs, please :)

Reply:
Huh, after removing these 2 lines, it seems that prettier is not happy. Its needs to be re-run.